src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -458,7 +486,10 @@ ServerHandlerImp::processSession(
                 jv,
                 {is->user(), is->forwarded_for()}};
 
+            auto start = std::chrono::system_clock::now();
             RPC::doCommand(context, jr[jss::result]);
+            auto end = std::chrono::system_clock::now();
+            logDuration(jv, end - start, m_journal);
```

Comment:
If this pattern is used in more places, it might be worth investing in a helper class that uses RAII in the style of `scope_guard`.